packages/components/src/Link/Link.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { PropsWithChildren } from "react";
+
+interface LinkProps {
+  readonly ariaLabel?: string;
+  readonly ariaExpanded?: boolean;
+  readonly external?: boolean;
+  readonly url: string;
```

Comment:
Small nitpick - consider moving this line for `url` to the top, since it is the only required property.